responses/en/HassListAddItem.yaml

```diff
@@ -0,0 +1,5 @@
+language: "en"
+responses:
+  intents:
+    HassListAddItem:
+      item_added: "Added {{ slots.item }} to {{ slots.name }}"
```

Comment:
Maybe this is a bit too much, according to the [style guide](https://developers.home-assistant.io/docs/voice/intent-recognition/style-guide)? I'd shorten it to `"Added {{slots.item}}"`